go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/wolfi-dev/wolfictl
 
-go 1.21
+go 1.21.0
```

Comment:
Why this change? Just curious

Reply:
Not sure! I just did `go mod tidy` and it added the patch version 🤔

Reply:
Okay! This is the closest I found for now: https://tip.golang.org/doc/go1.21#tools 🤷